firebase-firestore/ktx/src/main/kotlin/com/google/firebase/firestore/ktx/Firestore.kt

```diff
@@ -162,3 +172,67 @@ class FirebaseFirestoreKtxRegistrar : ComponentRegistrar {
     override fun getComponents(): List<Component<*>> =
             listOf(LibraryVersionComponent.create(LIBRARY_NAME, BuildConfig.VERSION_NAME))
 }
+
+/**
+ * Transforms a [DocumentReference] into a coroutine [Flow]
+ *
+ * **Backpressure handling**: by default this method conflates items. If the consumer isn't fast enough,
+ * it might miss some values but is always guaranteed to get the latest value. Use [bufferCapacity]
+ * to change that behaviour
```

Comment:
```suggestion
 * Starts listening to the document referenced by this `DocumentReference` with the given options and emits its values via a [Flow].
 *
 * When a collection of the returned flow starts, an EventListener will be attached.
 * When a collection completes, the listener will be removed.
 *
 * Backpressure: by default the returned flow is conflated. If the consumer isn't fast enough,
 * it might miss some values but is always guaranteed to get the latest value. Use the [bufferCapacity] parameter
 * to change that behaviour.
```